swift/perfect/Package.swift

```diff
@@ -4,9 +4,9 @@ import PackageDescription
 let package = Package(
     name: "server",
     dependencies: [
-        .package(url: "https://github.com/PerfectlySoft/Perfect-HTTPServer.git", "3.0.0"..<"3.1.0")
+        .package(url: "https://github.com/PerfectlySoft/Perfect", "3.1.0"..<"3.2.0")
     ],
     targets: [
-        .target(name: "server", dependencies: ["PerfectHTTPServer"], path: ".")
+        .target(name: "server", dependencies: ["Perfect"], path: ".")
```

Comment:
```suggestion
        .target(name: "server", dependencies: ["PerfectLib"], path: ".")
```

As far as I can tell Perfect calls their library package `PerfectLib` instead of `Perfect`. See https://github.com/PerfectlySoft/Perfect/blob/master/Package.swift#L25